service/generators.gradle

```diff
@@ -1,11 +1,9 @@
 dependencies {
-	implementation 'io.swagger.core.v3:swagger-annotations:2.2.0'
 	runtimeOnly 'org.webjars.npm:swagger-ui-dist:4.9.0'
-	swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.35'
 
-	// Needed by swagger codegen
-	implementation 'javax.validation:validation-api:2.0.1.Final'
-	implementation 'javax.annotation:javax.annotation-api:1.3.2'
+	implementation 'jakarta.annotation:jakarta.annotation-api:2.1.1'
+	implementation 'io.swagger.core.v3:swagger-annotations:2.2.12'
+	swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.45'
```

Comment:
These versions are different from those in `client/swagger.gradle`, is that intentional?

If you want to use the same versions in both, you should configure them in one of the conventions subprojects (like java-common-conventions) to avoid having to manually keep all the version numbers in sync.

Reply:
The server codegen can be Spring Boot 3, but the client needs to use older versions to be compatible with other services using Spring Boot 2

Reply:
Can the Spring Boot 3 service use a Spring Boot 2 codegen?
Or is it really that we need _**everything**_ to change at once?

Reply:
Yes, it can do that, we just need to provide the right generators, which is what this is doing. Its been tested with WSM and verified to work.